inprocess/src/test/java/io/grpc/inprocess/InProcessTransportTest.java

```diff
@@ -163,11 +178,62 @@ public Listener<Void> startCall(ServerCall<Void, Void> call, Metadata headers) {
                     .build();
     ClientCall<Void,Void> call = channel.newCall(nonMatchMethod, CallOptions.DEFAULT);
     try {
-      ClientCalls.futureUnaryCall(call, null).get(5, TimeUnit.SECONDS);
+      ClientCalls.futureUnaryCall(call, null).get(TIMEOUT_MS, TimeUnit.MILLISECONDS);
       fail("Call should fail.");
     } catch (ExecutionException ex) {
       StatusRuntimeException s = (StatusRuntimeException)ex.getCause();
       assertEquals(Code.UNIMPLEMENTED, s.getStatus().getCode());
     }
   }
+
+  @Ignore
```

Comment:
Is the @Ignore because it isn't working?

Reply:
This test is working. But this test is also running when AnonymousInProcessTransportTest runs and then fails because Anonymous.. creates its own transport. What I can do is override this test method (basicStreamInProcess) in AnonymousInProcessTransportTest and then @Ignore it. Solution works but is this the right approach? If yes then I also wanted to check with unit test's name `basicStreamInProcess`.

Reply:
You are using `newClientTransportWithAssumedMessageSize`, so you could just override that in AnonymousInProcessTransportTest so that it uses `address`